scripts/install_superclaude.sh

```
@@ -0,0 +1,229 @@
+            UV_INSTALLED=true
+            echo "✅ Homebrew経由でuvをインストールしました"
+        fi
+    fi
+
+    # 2. パッケージマネージャーが失敗した場合、ハッシュ検証付きダウンロードを実行
+    if [ "$UV_INSTALLED" = false ]; then
+        echo "🔐 セキュアダウンロード（ハッシュ検証付き）を実行中..."
+
+        # 一時ディレクトリ作成
+        UV_TEMP_DIR=$(mktemp -d)
+        trap "rm -rf '$UV_TEMP_DIR'" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)